docker/server/Dockerfile

```diff
@@ -16,19 +16,29 @@ RUN apt-get update \
     && env DEBIAN_FRONTEND=noninteractive \
         apt-get install --allow-unauthenticated --yes --no-install-recommends \
             clickhouse-common-static=$version \
+            clickhouse-client=$version \
             clickhouse-server=$version \
             libgcc-7-dev \
```

Comment:
why is it needed?

Reply:
For runtime code generation.
BTW, we only need "startup files": `crti.o`, etc.

Reply:
Better to provide these files in package.